src/style.css

```diff
@@ -1,3 +1,63 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  --color-blue: #00acdc;
+  --gap: 20px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
```

Comment:
Setting `width` and `height` on inline element is redundant. Try turning off those styles in browser dev tools.

Reply:
Yeah, it is not necessary